### DISCUSSION TITLE
Mirror list update request: eScience Center, Nanjing University （NJU）

### DISCUSSION THREAD

**Discussion by LadderOperator** [termux/termux-packages#7378] [General]
main deb https://mirror.nju.edu.cn/termux/apt/termux-main stable main
games deb https://mirror.nju.edu.cn/termux/apt/termux-games games stable
root deb https://mirror.nju.edu.cn/termux/apt/termux-root root stable
science deb https://mirror.nju.edu.cn/termux/apt/termux-science science stable
unstable deb https://mirror.nju.edu.cn/termux/apt/termux-unstable/ unstable main
x11 deb https://mirror.nju.edu.cn/termux/apt/termux-x11/ x11 main

Since this is neither a "Package Request" or  "Bug Report", I have to post here...

Thanks!

**Comment by ghost** [comment]
Thanks, it is added now.
